concert_demo_rapps/rapps/p2p_delivery/p2p_delivery.interface

```diff
@@ -0,0 +1,21 @@
+subscribers: 
+  - name: map
+    type: nav_msgs/OccupancyGrid
+  - name: map_metadata
+    type: nav_msgs/MapMetaData
+  - name: table_pose_list
+    type: yocs_msgs/TableList
+  - name: marker_pose_list
```

Comment:
This should be `global_marker_list` as we discussed in google doc and https://github.com/robotics-in-concert/rocon_demo_msgs/issues/5